docs/README.md

```diff
@@ -135,17 +137,16 @@ mywire.bom.tsv    BOM (bill of materials) as tab-separated text file
 mywire.html       HTML page with wiring diagram and BOM embedded
 ```
 
-#### Command line options
-
-- `--prepend-file <FILE>` to prepend an additional YAML file. Useful for part libraries and templates shared among multiple cables/harnesses.
-- `-o <OUTPUT>` or `--output_file <OUTPUT>` to generate output files with a name different from the input file.
-- `-V` or `--version` to display the WireViz version.
-- `-h` or `--help` to see a summary of the usage help text.
-
+Wildcars in the file path are also supported to process multiple files at once, e.g.:
+```
+$ wireviz ~/path/to/files/*.yml
+```
 
-### Syntax description
+To see how to specify the output formats, as well as additional options, run:
 
-A description of the WireViz YAML input syntax can be found [here](syntax.md).
+```
+$ wireviz --help
```

Comment:
You might want to mention both `-h` and `--help` alternatives (or only mention `-h`), but it might be equally good to leave it unchanged (maybe the long option is easier to remember for the inexperienced user).

Reply:
Let's leave as is for simplicity.